### PR TITLE
Updates for 389-ds-base-2.4.4 Fedora 39 dev release

### PR DESCRIPTION
**Pull request by jchapma**

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -62,16 +62,13 @@ If you're new to the project you should start with our [getting started](docs/38
 
 ## Community
 
-To get in touch with us, you can try to reach us on **IRC** at **\#389** on **irc.libera.chat** or on one of our [mailing lists](docs/389ds/mailing-lists.html).
+To get in touch with us, you can reach us on slack at #team-ldap or on one of our [mailing lists](docs/389ds/mailing-lists.html).
```
Comment (tbordaz) [inline index.md:65]: I am unsure slack channel would be available for the community

**Review by tbordaz — COMMENTED** [review]
Very nice. Just a minor question.

File: index.md (hunk 1 above)
Comment (vashirov) [reply index.md:65]: Right, this is internal instance. We still have presence on libera.chat

Comment (jchapma) [reply index.md:65]: Gulp, my bad. I thought the opposite for both slack and libera.chat

**Pull request merged**